nsxt/resource_nsxt_policy_tier0_gateway_gm_test.go

```diff
@@ -0,0 +1,197 @@
+/* Copyright © 2019 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"testing"
+)
+
+var testAccGmGatewayIntersiteSubnet = "10.10.2.0/24"
+
+// NOTE: This test assumes single edge cluster on both sites
+func TestAccResourceNsxtPolicyTier0Gateway_globalManager(t *testing.T) {
+	testResourceName := "nsxt_policy_tier0_gateway.test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
```

Comment:
Shall we also add "testAccOnlyGlobalManager(t)" here?

Reply:
You're right, just counting on SITE definition is not enough. Thanks